google/cloud/storage/object_stream.h

```diff
@@ -343,6 +343,18 @@ class ObjectWriteStream : public std::basic_ostream<char> {
    */
   void Suspend() &&;
 
+  /**
+   * Returns the status of partial errors.
+   *
+   * Application may write multiple times before closing the stream, this
+   * function gives the capability to find out status even before stream
+   * closure.
+   *
+   * This function is different than `metadata()` as calling `metadata()`
+   * before Close() is undefined.
+   */
+  Status const& last_status() const { return buf_->last_status(); }
```

Comment:
Probably will need to return by value here.

Reply:
fixed.